packages/fxa-settings/src/components/Settings/FlowRecoveryKeyConfirmPwd/index.test.tsx

```diff
@@ -0,0 +1,120 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { act, fireEvent, screen } from '@testing-library/react';
+import { logViewEvent } from '../../../lib/metrics';
+import FlowRecoveryKeyConfirmPwd from './';
+import {
+  mockAppContext,
+  MOCK_ACCOUNT,
+  renderWithRouter,
+} from '../../../models/mocks';
+import { Account, AppContext } from '../../../models';
+import { typeByLabelText } from '../../../lib/test-utils';
+
+const localizedBackButtonTitle = 'Back to settings';
+const localizedPageTitle = 'Account Recovery Key';
+const navigateForward = jest.fn();
+const navigateBackward = jest.fn();
+const viewName = 'settings.account-recovery';
+
+jest.mock('../../../lib/metrics', () => ({
+  usePageViewEvent: jest.fn(),
+  logViewEvent: jest.fn(),
+}));
+
+jest.mock('base32-encode', () =>
+  jest.fn().mockReturnValue('00000000000000000000000000000000')
+);
+
+const setFormattedRecoveryKey = jest.fn();
+
+const accountWithSuccess = {
+  ...MOCK_ACCOUNT,
+  createRecoveryKey: jest.fn().mockResolvedValue(new Uint8Array(20)),
+} as unknown as Account;
+
+const renderFlowPage = (account: Account) => {
+  window.URL.createObjectURL = jest.fn();
+  renderWithRouter(
+    <AppContext.Provider value={mockAppContext({ account })}>
+      <FlowRecoveryKeyConfirmPwd
+        {...{
+          localizedBackButtonTitle,
+          localizedPageTitle,
+          navigateForward,
+          navigateBackward,
+          setFormattedRecoveryKey,
+          viewName,
+        }}
+      />
+    </AppContext.Provider>
+  );
+};
+
+describe('FlowRecoveryKeyConfirmPwd', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders as expected', async () => {
+    renderFlowPage(accountWithSuccess);
+    expect(
+      screen.getByRole('heading', {
+        name: 'Enter your password again to get started',
+      })
+    ).toBeInTheDocument();
+    expect(screen.getByLabelText('Enter your password')).toBeInTheDocument();
+    expect(
+      screen.getByRole('button', { name: 'Create recovery key' })
+    ).toBeInTheDocument();
```

Comment:
Did we decide we stylistically preferred to keep `toBeInTheDocument` even if it's redundant by just using `getBy`? Idk. I might continue nitpicking this in PRs unless we say otherwise because it's extra lines to double test 😅

Reply:
I'm not too opinionated on this - I like that `toBeInTheDocument` is explicit, but also like that omitting it is more concise. I've updated the tests to remove `toBeInTheDocument`.